tests/tools/onert_train/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ list(APPEND ONERT_TRAIN_SRCS "src/nnfw_util.cc")
 list(APPEND ONERT_TRAIN_SRCS "src/randomgen.cc")
 list(APPEND ONERT_TRAIN_SRCS "src/rawformatter.cc")
 list(APPEND ONERT_TRAIN_SRCS "src/rawdataloader.cc")
+list(APPEND ONERT_TRAIN_SRCS "src/metric.cc")
```

Comment:
(beyond this PR) @jyoungyun What about to use glob? instead of adding each file. It is the usual way we use under `/runtime/onert`. Previously, for `onert_run`, I've appended all the files in `onert_run`. But `onert_train` has more files, and it is likely to have more files.

Reply:
We can use `glob` to collect all cc files. I will fix it later.